server/routes/appointments/create-and-edit/handlers/schedule.ts

```diff
@@ -124,5 +126,37 @@ export default class ScheduleRoutes {
     }
 
     res.redirect(`../../schedule${req.query.preserveHistory ? '?preserveHistory=true' : ''}`)
+
+    this.trackAppointmentJourneyMode(res, req)
+  }
+
+  CHANGE = async (req: Request, res: Response): Promise<void> => {
+    const { property, preserveHistory } = req.query
+    const propertyAsString: string = property as string
+    const properties = {
+      username: res.locals.user.username,
+      prisonCode: res.locals.user.activeCaseLoadId,
+      propertyName: propertyAsString,
```

Comment:
This one is my bad for not adding it to the original ticket and then not explaining the addition fully. It's adding `appointmentJourneyMode: req.session.appointmentJourney.mode` into these properties rather than adding the new `trackAppointmentJourneyMode` function and 'SAA-Appointments-Appointment-Journey-Mode' custom event